external/googleapi/models.py

```diff
@@ -0,0 +1,134 @@
+import attr
+from typing import List
+
+
+@attr.s
+class GeocodeAddressComponent(object):
+    long_name = attr.ib()
+    short_name = attr.ib()
+    types: List[str] = attr.ib(factory=list)
+
+    @classmethod
+    def list_converter(cls, list):
+        return [GeocodeAddressComponent.get_converter(li) for li in list]
+
+    @classmethod
+    def get_converter(cls, ob):
```

Comment:
Optional: For me, `get_converter` part seems to be a bit complicated and might be confusing in the future.

I think `get_converter` does not return a converter but it's converter itself. How about other name like`from_any`? 

Or we can just write `def __init__(self, ob):` and pass `GeocodeAddressComponent` instead of `GeocodeAddressComponent.get_converter`.  I'm not sure whether the default initializer given by attr.s will go away or not when we write our own initializer

Reply:
good point! I'll try it out and see which works.

Reply:
so it looks like attrs advises against writing your own __init__ methods, so I'll keep the `@classmethod` and change the name.